config.json

```diff
@@ -121,7 +121,6 @@
       "unlocked_by": null,
       "difficulty": 3,
       "topics": [
-        "arithmetic",
```

Comment:
I'm a bit unsure whether we remove arithmetic here. I think that it's not really mathy, but I'll admit that there's arithmetic involved. I'm tempted to leave it off per the discussion in https://github.com/exercism/exercism.io/issues/4110

Reply:
It uses floating point arithmetic, which is a bit special in OCaml (you need to use floating point specific operators as there's no overloading). So floating_point might be worth adding to this list.

Reply:
I agree! I'll update this one to say 'floating_point'. Thanks!

Reply:
@stevejb71 It already has `floats`. Should we go with `floats` or `floating_point`?

Reply:
I see the topics.txt file has floating_point_numbers, so I think we should go with that.

Reply:
One thing to be aware of is that in the UI long names get truncated. I'd lean towards `floats` (and would suggest updating `topics.txt`, too).

Reply:
Might be good to add a rule for truncation in the config linter then.

I'm ok with floats.

Reply:
Ok, cool, thanks!